lfucg-exactions/client/js/components/AccountLedgerForm.js

```diff
@@ -324,9 +335,9 @@ function mapDispatchToProps(dispatch, params) {
             dispatch(formInit());
             dispatch(getAccounts());
             dispatch(getAgreements());
-            dispatch(getLots());
-            dispatch(getPlats());
             if (selectedAccountLedger) {
+                dispatch(getPlats());
```

Comment:
You need to get plats if it is an existing account ledger (they are always based on the lot).